application/controllers/classification_type_controller.py

```diff
@@ -0,0 +1,57 @@
+from structlog import get_logger
+from sqlalchemy.exc import SQLAlchemyError
+
+from application.models.classification_type import ClassificationType
+from application.utils.exceptions import InvalidClassificationType, DatabaseError
+from application.utils.database import db
+logger = get_logger()
+
+
+def get_classification(classification_type):
+    try:
+        classification = db.session.query(ClassificationType)\
```

Comment:
Does this actually throw a SQLAlchemyError?

Reply:
It's the base exception for all the sqlalchemy exceptions, I had it more as a catch all for any possible database error (connection etc)